projects/aca-folder-rules/src/lib/rule-details/conditions/rule-simple-condition.ui-component.ts

```diff
@@ -92,6 +100,10 @@ export class RuleSimpleConditionUiComponent implements ControlValueAccessor, OnD
     return this.form.get('comparator');
   }
 
+  get valueControl(): AbstractControl {
```

Comment:
this can be private as you don't use it outside this class, also "valueControl" name is misleading, as inside you call it "parameter". Better use "parameterControl" or something like that.

Reply:
getter made private and name changed